cinema/src/main/java/com/greg/cinema/service/ScreeningService.java

```diff
@@ -0,0 +1,13 @@
+package com.greg.cinema.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.greg.cinema.entity.ScreeningEntity;
+
+public interface ScreeningService {
+	
+	public List<ScreeningEntity> listScreenings(Optional<String> time);
```

Comment:
a jak będziemy mieli naprawdę duż seansów to co może pójść nie tak?

Reply:
Wtedy można by dodać paginację :)